From: "Jason A. Donenfeld" <Jason@zx2c4.com>
Date: Tue, 21 Jan 2020 16:54:56 +0100
Subject: [PATCH] Makefile: do not use git to get version number

Debian prefers to stick with the one in the source code.

Signed-off-by: Jason A. Donenfeld <Jason@zx2c4.com>
---
 src/Makefile | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/src/Makefile b/src/Makefile
index c20bfd3..729f22a 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -20,19 +20,17 @@ ifneq ($(V),1)
 MAKEFLAGS += --no-print-directory
 endif
 
-WIREGUARD_VERSION = $(patsubst v%,%,$(shell GIT_CEILING_DIRECTORIES="$(PWD)/../.." git describe --dirty 2>/dev/null))
-
 module:
-	@$(MAKE) -C $(KERNELDIR) M=$(PWD) WIREGUARD_VERSION="$(WIREGUARD_VERSION)" modules
+	@$(MAKE) -C $(KERNELDIR) M=$(PWD) modules
 
 module-debug:
-	@$(MAKE) -C $(KERNELDIR) M=$(PWD) V=1 CONFIG_WIREGUARD_DEBUG=y WIREGUARD_VERSION="$(WIREGUARD_VERSION)" modules
+	@$(MAKE) -C $(KERNELDIR) M=$(PWD) V=1 CONFIG_WIREGUARD_DEBUG=y modules
 
 clean:
 	@$(MAKE) -C $(KERNELDIR) M=$(PWD) clean
 
 module-install:
-	@$(MAKE) -C $(KERNELDIR) M=$(PWD) WIREGUARD_VERSION="$(WIREGUARD_VERSION)" modules_install
+	@$(MAKE) -C $(KERNELDIR) M=$(PWD) modules_install
 	$(DEPMOD) -b "$(DEPMODBASEDIR)" -a $(KERNELRELEASE)
 
 install: module-install
